assets/base/i18n/en.json

```diff
@@ -199,6 +199,15 @@
   "global_threads.markAllRead.markRead": "Mark read",
   "global_threads.markAllRead.message": "This will clear any unread status for all of your threads shown here",
   "global_threads.markAllRead.title": "Are you sure you want to mark all threads as read?",
+  "global_threads.options.copy_link": "Copy Link",
+  "global_threads.options.mark_as_read": "Mark as Read",
+  "global_threads.options.mark_as_unread": "Mark as Unread",
+  "global_threads.options.open_in_channel": "Open in Channel",
+  "global_threads.options.reply": "Reply",
+  "global_threads.options.save": "Save",
+  "global_threads.options.title": "THREAD ACTIONS",
+  "global_threads.options.unfollow": "Unfollow Thread",
+  "global_threads.options.unsave": "Unsave",
```

Comment:
is there a reason to duplicate most of these?

Reply:
I thought maintaining them separately makes it easy to change the language in future like from "Save" to "Save thread" or something as threads are different from posts.

Reply:
I get the reasoning but I wouldn't create new strings if there is no real need, in the future if things changed, we can look into adding a few new strings